Automattic/wp-calypso
# Posts / State: Add post counts to global state

This pull request seeks to implement post counts behaviors in the global Redux application state. This is planned to be used as part of the focus on Custom Post Types, specifically in displaying counts for each status for a post type in the list view filter bar.

__Testing instructions:__

The post counts state is not used yet, but you should verify that the inclusion of the new `state.posts.counts` subtree does not introduce regressions to the application.

Ensure that Mocha tests pass by running `npm run test-client`.

__Open questions:__

- The state is implemented using WP-API undocumented post types endpoint. @timmyc @nylen : Do you agree that this is the correct approach to be using here, as you've started to roll out in the editor (#4107)?
